File: 2189/2189.py
# bee 2189 - Quermesse
# Alonso Martins
# 18/03/2024
t = 0
while True:
	N = int(input())
	
	if N == 0:
		break
	t += 1
	
	ent = list(map(int, input().split()))
	
	i = 0
	
	while ent[i] != i + 1:
		i += 1
	
	print("Teste", t)	
	print(i + 1)
	print()

File: 2189/2189.c
// bee 2189 - Quermesse
// Alonso Martins
// 18/03/2024
#include <stdio.h>
#include <stdlib.h>

int main()
{
    int t, N, i, *ent;

    t = 0;

    while (scanf("%d", &N) && N)
    {
        t += 1;

        ent = (int *)malloc(N * sizeof(int));
        for (i = 0; i < N; i ++)
            scanf("%d", &ent[i]);
        
        i = 0;

        while (ent[i] != i + 1)
            i++;

    	printf("Teste %d\n%d\n\n", t, i + 1);
    }
    return 0;
}
